### ISSUE TITLE
[Feature Request] Option: Reset to today when Itsycal is closed

### ISSUE THREAD

**Issue by navinthomas** [sfsam/Itsycal#225]
### Current Behaviour

- Itsycal opens with today as the currently selected date when the app is initially launched
- Itsycal opens with the previously selected date when the app remains running

### Suggested Behaviour

* Add an option to make Itsycal open always with today as the currently selected date
  *  Preferably as an option in settings

### Reason for Suggestion

Occasionally, I unknowingly miss out on seeing today's events. This happens when I had selected some other date in Itsycal previously and then forgot to reset it back to today. By the time I realize what happened, it is too late. This could have drastic consequences for time sensitive and critical events.

**Comment by sfsam** [comment]
Thanks for the suggestion. The current behavior is to revert to today if the calendar has been hidden for 10 minutes or more. Otherwise, it will show the last selected day.

**Comment by navinthomas** [comment]
Yeah, I think having a configurable option in the settings page would be nice. Also, I wasn't aware that the current behaviour is to reset to today after the app has been hidden for 10 minutes. Thanks for info :)
